app/adapters/fetchallnews/client.rb

```diff
@@ -0,0 +1,67 @@
+module Fetchallnews
+  class Client
+    include HTTParty
+
+    RequestResult = Struct.new(:succes?, :errors, :data)
+
+    def send_request
+      RequestResult.new(success?, errors, data)
+    end
+
+    def success?
+      mediastack_response.success? || currentsapi_response.success?
+    end
+
+    def errors
+      [mediastack_response['error'], currentsapi_response['msg']].flatten
```

Comment:
I think it should be written like this:
`[mediastack_response['error'], currentsapi_response['msg']].compact.flatten`
in case if only one api will return error and the second wil return `nil`